src/components/Project/Project.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
+
+import {
+  Box,
+  Card,
+  CardContent,
+  Link,
+  Typography,
+} from '@mui/material';
+import { } from 'react';
+
+interface Link {
+  title: string;
+  url: string;
+}
+
+interface ProjectProps {
```

Comment:
I think we should move this to a separate file.

Not sure if this is the right name. Maybe Project?

Reply:
Addressed in [bbe4e62](https://github.com/OpenBCca/openbc-web/pull/5/commits/bbe4e622e4e629cb656fed7d103a4d736bd02764)